README.md

```diff
@@ -0,0 +1,91 @@
+# README
+
+## Introduction
+
+This script is an example implementation of a shielded transfer using the Light Protocol and **Solana** blockchain. It demonstrates how to initialize a **Solana** wallet, request an airdrop of SOL tokens, set up a test relayer, perform a shield operation, transfer tokens to a recipient, and retrieve transaction information.
+
+## Prerequisites
+
+Before running this script, ensure that you have the following prerequisites:
+
+1. **Node**.js and **npm** installed on your machine.
+2. A **Solana** wallet file (JSON format) located at `~/.config/solana/id.json`. If the file doesn't exist, the script will generate a new wallet for testing purposes.
```

Comment:
this wallet will not be generated, just the lookup table

Reply:
what about this? 

```
  const solanaWallet = anchor.web3.Keypair.generate(); // Replace this with your user's Solana wallet
```

Reply:
Good point, it looks like the wallet file is not being used and the path is just set to make the anchor provider happy.
A new wallet will always be generated, not if the file doesn't exist :)

You can try what happens when the file doesn't exist, I don't remember.